source/python/idetect/nlp_models/relevance/relevance.py

```diff
@@ -0,0 +1,205 @@
+import os
+import requests
+import gensim
+import pandas as pd
+import numpy as np
+from nltk.tokenize import WordPunctTokenizer
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.pipeline import Pipeline
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.externals import joblib
+
+from idetect.model import Relevance
+
+class RelevanceModel(object):
+    def __init__(self, model_path=None):
+        self.model = self.load_model(model_path=model_path)
+
+    def load_model(self, model_path=None):
+        if model_path and os.path.isfile(model_path):
```

Comment:
Or RelevanceModel and CategoryModel could both be instances of class DownloadableModel initialized with different `model_path` and `url` values.